src/lib/features/feature-toggle/feature-toggle-strategies-store.ts

```diff
@@ -119,6 +120,10 @@ const sortEnvironments = (overview: IFeatureOverview) => {
     }));
 };
 
+function mergeAll<T>(objects: Partial<T>[]): T {
+    return merge.all<T>(objects.filter((i) => i));
+}
+
```

Comment:
hmm, we already have this in create-config as well. Should we extract it to a helper? Where to put it, types/index ? Create a merger.ts file?